### PR TITLE
Next js course by hitesh choudhary

### PR DESCRIPTION
**Pull request by MOHDNEHALKHAN**
## What does this PR do?
Add a new  next.js course in Hindi by Hitesh Choudhary

### Description
there is a playlist of the course covering all concepts with hands-on project practise

### Why is this valuable (or not)?
for beginners

### How do we know it's really free?
Yes , it is available free on YouTube 

### For book lists, is it a book? For course lists, is it a course? etc.
yes it is a course

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by MOHDNEHALKHAN** [comment]
@eshellman , accept by new PR under hacktoberfest 2024

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -472,6 +472,7 @@
 * [Next JS Tutorial for Beginners in Hindi](https://www.youtube.com/playlist?list=PLZjjdd9-SJS2ZvI4ct5Qtkje_Vdb5O_KM) - ILive4Coding
 * [Next js tutorial in Hindi](https://www.youtube.com/playlist?list=PL8p2I9GklV44sj_Ikp8jQSvwD-m9htnHT) - Code Step By Step
 * [Next.js Tutorials for Beginners in Hindi](https://www.youtube.com/playlist?list=PLu0W_9lII9agtWvR_TZdb_r0dNI8-lDwG) - CodeWithHarry
+* [NextJS Chai aur full stack ](https://www.youtube.com/playlist?list=PLu71SKxNbfoBAaWGtn9GA2PTw0HO0tXzq) - Chai aur Code
```
Comment (eshellman) [inline courses/free-courses-hi.md:475]:
```suggestion
* [NextJS Chai aur full stack](https://www.youtube.com/playlist?list=PLu71SKxNbfoBAaWGtn9GA2PTw0HO0tXzq) - Chai aur Code
```

Comment (MOHDNEHALKHAN) [reply courses/free-courses-hi.md:475]: I will do it today 👍🏻

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**